### DISCUSSION TITLE
Blog post about: Composition over Inheritance

### DISCUSSION THREAD

**Discussion by Rodien** [2sic/2sxc#3126] [General]
@iJungleboy 

I would like to read a piece about the following bullet point of the latest 2SXC release "Lots of internal refactoring for Composition over Inheritance," discussing the problems encountered, the approach taken, and the considerations when working with composition. Furthermore, I'm interested in learning about the problems observed when using composition and the corresponding solutions or approaches adopted.

If you have time to write a blog post it would be very much appreciated.

**Comment by iJungleboy** [comment]
Hi @Rodien 

I would love to, but I simply don't have time for that, sorry :(
